src/Components/GithubCard.css

```diff
@@ -0,0 +1,51 @@
+.box-header {
```

Comment:
I realize some of these are specific to the github controls, but are there styles you could extract that will end up being common to all the controls for the different sites? Might make sense to have a control panel component that can change for each site rather than building it into the card.

Reply:
need to discuss this